api/app/serializers/spree/v2/storefront/state_serializer.rb

```diff
@@ -4,7 +4,7 @@ module Storefront
       class StateSerializer < BaseSerializer
         set_type :state
 
-        attributes :abbr, :name
+        attributes :id, :abbr, :name
```

Comment:
@MatthewKennedy you don't need to add `id` as an attribute, it's added automatically by JSON API in a higher node level

![Screen Shot 2019-08-05 at 12 22 33 PM](https://user-images.githubusercontent.com/55154/62457763-bcecb280-b77b-11e9-84c7-8b98c15bc4a7.png)